frontend/src/containers/UserApplication/index.jsx

```diff
@@ -55,11 +53,15 @@ const UserApplication = ({
             <SmallDescriptionWrapper>
               <div>
                 <SmallDescription> Sendt </SmallDescription>
-                <Moment format="dddd Do MMMM, \k\l. HH:mm">{created_at}</Moment>
+                <FormatTime format="EEEE dd MMMM, \k\l. HH:mm">
+                  {created_at}
+                </FormatTime>
               </div>
               <div>
                 <SmallDescription> Oppdatert </SmallDescription>
-                <Moment format="dddd Do MMMM, \k\l. HH:mm">{updated_at}</Moment>
+                <FormatTime format="EEEE dd MMMM, \k\l. HH:mm">
+                  {updated_at}
+                </FormatTime>
```

Comment:
It seems the escaping sequence has changed somewhat (screenshot from the group-representative view, that part is fine from the superuser view)
![image](https://user-images.githubusercontent.com/13599770/187073371-ff70c337-32b4-412d-a29d-10fac0753bbb.png)

Reply:
Yeah, seems I forgot to change it in some places :+1: